modules/user_data/variables.tf

```
@@ -96,12 +94,11 @@ variable "enable_metrics_collection" {
 }
 
 variable "extra_no_proxy" {
-  default     = null
   description = <<-EOD
   A list of hosts for which Terraform Enterprise will not use a proxy to access. The list must be a comma-separated
   string, like \".example.com,.example.org\".
   EOD
```

Great, thanks for incorporating this change now!

```suggestion
  description = "A list of hosts for which Terraform Enterprise will not use a proxy to access."
```